LocalSettings.php

```diff
@@ -430,9 +430,9 @@
 		'betawiki' => true,
 	],
 	'wgCentralAuthEnableUserMerge' => [
```

Comment:
If we removed the extension, there is no point for that config to be there.

Reply:
We aren't removing CentralAuth

Reply:
Nor does this PR remove it, it just disables it.

Reply:
I already mentioned this also in my comment above. The config should probably be removed since its disabled by default in REL1_37 and completely removed in master (1.38)